cfgov/regulations3k/jinja2/regulations3k/browse-regulation.html

```diff
@@ -1,7 +1,6 @@
 {% extends 'regulations3k/regulations3k-side-nav.html' %}
 
 {% import 'form_block.html' as form_block with context %}
-{% import 'organisms/featured-content.html' as featured_content with context %}
```

Comment:
Am I reading this correctly that this organism isn't actually used on this Regs3k page type? It doesn't seem to be one of the [supported block types](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L201)?

Reply:
That does seem correct to my eyes, as well. @willbarton or @contolini, can you confirm?

Reply:
Hmmm I bet we copied a template when we started regs3k and accidentally left some extraneous stuff.